### ISSUE TITLE
FR: Search Function for Tags

### ISSUE THREAD

**Issue by chrisgrieser** [alexobenauer/Cross-reference-Navigation-for-Obsidian#12]
I use hundreds of tags, the lack of a search function makes it quite tedious to find the right tag 😅 

SO a search function would indeed be very useful

**Comment by kmaustral** [comment]
Yes, this looks wonderfully promising, like a graph view for tags, but it's hopeless for me with so many tags that run off the screen.

**Comment by kadmy** [comment]
> Yes, this looks wonderfully promising, like a graph view for tags, but it's hopeless for me with so many tags that run off the screen.

If you combine tags into groups, it works fine
